functions/onprem/orborus/orborus.go

```diff
@@ -1404,10 +1445,119 @@ func main() {
 	//defer cleanup()
 
 	// Block until a signal is received
-	if isRunningInCluster() {
+	if shuffle.IsRunningInCluster() {
 		log.Printf("[INFO] Running inside k8s cluster")
 	}
 
+	if isKubernetes == "true" {
+		clientset, _, err := shuffle.GetKubernetesClient()
+		if err != nil {
+			log.Printf("[ERROR] Error getting kubernetes client:", err)
+			os.Exit(1)
```

Comment:
os.Exit is dangerous. Never use it when pushing to customers. Better to just return and hope it fixes itself

Reply:
left it because there is only 1 os.Exit and that is during the launch. if we can't create a k8s client at the beginning of the launch, there is a problem to be sorted out.